main.py

```diff
@@ -15,8 +15,18 @@ def hello():
 
 
 @app.route("/post/<int:post_id>")
+def get_post_no_title(post_id):

```

Comment:
This won't scale well. Making 2 DB requests per page load (if the page is accessed via a "short link" - just the post ID) will be very inefficient at scale. The way it was is more efficient.